packages/ui-library/src/components/button-group/index.stories.ts

```diff
@@ -57,7 +57,7 @@ export default {
     docs: {
       description: {
         component: `<markdown>
-        Button Group allows users to select one option from the group. They are visually cohesive, meaning they share a similar appearance in terms of size, style, and often color. This consistency helps users quickly identify that these buttons are related. The Button Group component utilizes the slot element, rendering multiple Text Button or Icon Button components, or the combination of both. For more information have a look at the [Text Button](?path=/docs/design-system-web-components-actions-buttons-text-button--docs) and the [Icon Button](?path=/docs/design-system-web-components-actions-buttons-icon-button--docs) components.
+        Button Group allows users to select one option from the group. They are visually cohesive, meaning they share a similar appearance in terms of size, style, and often color. This consistency helps users quickly identify that these buttons are related. The Button Group component utilizes the slot element, rendering multiple Button Text or Button Icon components, or the combination of both. For more information have a look at the [Button Text](?path=/docs/design-system-web-components-actions-buttons-button-text--docs) and the [Button Icon](?path=/docs/design-system-web-components-actions-buttons-button-icon--docs) components.
```

Comment:
The links to the button text and button icon are not working. I suppose his has been the case since we got rid of the component grouping within #859 . Could you please fix the links here:

Link to Button Text: `?path=/docs/components-button-text--docs`
Link to Button Icon: `?path=/docs/components-button-icon--docs`